packages/ses/README.md

```diff
@@ -445,6 +520,10 @@ The compartment will call `execute` with:
 method of third-party static module records to return promises, to support
 top-level await.
 
+:warning: The virtual module source interface does not yet agree with the
+[XS](https://www.moddable.com/hardening-xs) implementation of [Hardened
+JavaScript](https://hardenedjs.org/).
```

Comment:
I what way? (i'm necessarily suggesting answering inline, depending on the answer. But I would like to know.)

Reply:
We implemented virtual module sources that are identified by `exports`, `imports`, `reexports`, and `execute` properties, and pass to `execute` a mutable facet of the module exports namespace, the `compartment`, and the resolutions for all of the `imports` such that it can call `compartment.importNow(resolutions[importSpecifier])` to get a namespace for its dependencies.

Moddable implemented virtual module sources that are identified by `bindings` and `execute` properties, where `execute` receives a mutable facet of the module imports namespace (for both imports and exports), dynamic `import`, and `importMeta`.

The design here is in flux with the Module Harmony folks, but between us, Moddable’s design is really nice and I’d like to implement it eventually.

Reply:
Thanks. Resolving.